bert/common/helper.py

```diff
@@ -9,7 +9,7 @@
 
 _bert_small = dict(
     seq_length=512,
-    vocab_size=32400,
+    vocab_size=50304,

```

Comment:
the setting of vocab_size must be divisible by TP size in 2D, 2.5D, and 3D.
@binmakeswell @kurisusnowdeng @FrankLeeeee  I do not think users can accept this constrains.